### DISCUSSION TITLE
4/10 - Discussion with Prof

### DISCUSSION THREAD

**Discussion by willschenk** [ellenguyen/CIS4496_EY#14] [General]
DEMO FEEDBACK 04/10
- should look into LBP features, batch normalization, brightness (doesn't make sense to hard code params to a fixed brightness)
- better or worse than hierarchical model (sort of a decision tree)? should try both to see whatever is better
- Microsoft PNI for hyperparameter tuning
